app/main.py

```diff
@@ -1,8 +1,52 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        if not (1 <= comfort_class <= 7):
+            raise ValueError("Comfort class must be between 1 and 7")
+        if not (1 <= clean_mark <= 10):
+            raise ValueError("Clean mark must be between 1 and 10")
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        if not (1.0 <= distance_from_city_center <= 10.0):
+            raise ValueError("Comfort class must be between 1 and 7")
+        if not (1.0 <= average_rating <= 5.0):
+            raise ValueError("Clean mark must be between 1 and 10")
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for single_car in cars:
+            if single_car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(single_car)
+                self.wash_single_car(single_car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: Car) -> float:
+        price = ((car.comfort_class
```

Comment:
Here you don't need to store price in a variable (since you don't reuse it)  - you can return straight away.